compiler/passes/convert-uast.cpp

```diff
@@ -965,8 +965,8 @@ struct Converter {
   }
 
   BlockStmt* visit(const uast::Break* node) {
-    const char* name = node->target() ? node->target()->name().c_str()
-                                      : nullptr;
+    const char* name = nullptr;
+    if (auto target = node->target()) name = astr(target->name().c_str());
```

Comment:
actually I added `const char* astr(UniqueString s);` earlier so you can just do `name = astr(target->name())`.